components/home-page/bal-action/action-bbe/working-with-databases.md

```diff
@@ -45,4 +45,3 @@ service / on new http:Listener(8080) {
         return album;
     }
 }
-```
```

Comment:
This should be added back

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6115.